lib/rspec/watchdog/slow_spec_formatter.rb

```diff
@@ -11,12 +11,13 @@ def initialize(output)
     @flaky_spec_detection = RSpec.configuration.flaky_spec_detection
     @watchdog_api_url = Rspec::Watchdog.config.watchdog_api_url
     @watchdog_api_token = Rspec::Watchdog.config.watchdog_api_token
+    @fast_specs_threshold = RSpec.configuration.fast_specs_threshold
```

Comment:
Should this be reading watchdog config?
```suggestion
    @fast_specs_threshold = Rspec::Watchdog.config.fast_specs_threshold
```
I get the following error when I try to use this formatted:
```An error occurred while loading ./spec/workers/attendease_business_unit_id_updater_job_spec.rb.
Failure/Error: config.add_formatter(SlowSpecFormatter)

NoMethodError:
  undefined method `fast_specs_threshold' for #<RSpec::Core::Configuration:0x00000001291bde90>
  ```